ui/src/containers/Search/index.tsx

```diff
@@ -18,11 +25,22 @@ const Search: React.FC = () => {
     }, 20);
   });
 
-  const { resources } = useMst();
+  const debounced = useDebouncedCallback(
```

Comment:
Could you add a comment here what this function does ??

Reply:
yes sure